app/upload/page.tsx

```
@@ -5,91 +5,362 @@ import { Input } from '@/components/ui/input';
 
-    const formData = new FormData();
-    formData.append('image', selectedFile!);
+  const handleSubmit = async (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    setIsUploading(true);
 
     try {
-      const resp = await fetch('/api/patient/upload/before', {
+      // Upload before image
+      const beforeFormData = new FormData();
+      beforeFormData.append('image', beforeFile!);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)